src/util/util.js

```diff
@@ -87,6 +103,28 @@ export function getBounds(points: Point[]): { min: Point, max: Point} {
     };
 }
 
+/**
+ * Returns the 2D square distance between an AABB defined by min and max and a point.
```

Comment:
To clearly indicate it's not the manhattan distance:

```suggestion
 * Returns the square of the 2D distance between an AABB defined by min and max and a point.
```